Solutions/NXLogAixAudit/Data Connectors/NXLogAixAudit.json

```diff
@@ -0,0 +1,110 @@
+        },
+        {
+            "description" : "Time chart of AIX Audit events per hour",
+            "query": "NXLog_parsed_AIX_Audit_view\n| where EventTime >= todatetime('2021-08-17')\n| where EventTime <  todatetime('2021-08-18')\n| summarize Count=count() by bin(EventTime, 1h)\n| render timechart title=\"AIX Audit events per hour\""
+        },
+        {
+            "description" : "AIX Audit Events per Second (EPS) Time Chart",
+            "query": "NXLog_parsed_AIX_Audit_view\n| where EventTime >= todatetime('2021-08-17 18:29')\n| where EventTime <  todatetime('2021-08-17 23:55')\n| summarize EPS=count() by bin(EventTime, 1s)\n| render timechart title=\"AIX Audit Events per Second (EPS)\""
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "AIX_Audit_CL",
+            "lastDataReceivedQuery": "NXLog_parsed_AIX_Audit_view            | summarize Time = max(TimeGenerated)            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "NXLog_parsed_AIX_Audit_view | summarize LastLogReceived = max(TimeGenerated) | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**NXLog_parsed_AIX_Audit_view**](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/NXLogAixAudit/Parsers/NXLogAixAudit.txt) which is deployed with the Azure Sentinel Solution.",

```

Comment:
Please update the parser link as https://aka.ms/sentinel-nxlogaixaudit-parser

Reply:
Resolved with commit [f6788f0](https://github.com/Azure/Azure-Sentinel/pull/2968/commits/f6788f0d4f7f3d1543d9db4b913bbd25cefc319f)